src/client/components/Menu/style.css

```diff
@@ -3,11 +3,23 @@
     list-style: none;
     margin: 0;
     display: flex;
+    transform:rotate(-20deg); 
+    -webkit-transform:rotate(-20deg); 
+    -moz-transform:rotate(-20deg); 
+    -o-transform:rotate(-20deg); 
```

Comment:
:medal_military: - dokładnie tego chciałem użyć by to obrócić później. Też myślałem o tym, by mieć proste logo i obrócić całość. Jeśli się nie da, trzeba będzie obracać elementy z osobna.

Reply:
I brakło ci spacji za dwukropkiem

Reply:
przy przeklejaniu mi uciekło :)
BYłem zbyt podniecony że mi działa :D